app/models/HowDisposed.scala

```diff
@@ -23,8 +23,8 @@ object HowDisposed {
 
   sealed trait HowDisposed
 
-  case object Sold extends WithName("sold") with HowDisposed
```

Comment:
When these values are written to mongodb, they are written as: `"_type": "models.HowDisposed.Sold"`.
If this is an easy fix, could you please fix this @narendrapaduchuri on your next ticket's branch?

Reply:
Sure Kornel, I will check and do in my next branch.